adds information about the cron log (#1418)
platformsh/platformsh-docs: docs/src/configuration/app/cron.md

If an application defines both a `web` instance and a `worker` instance, cron tasks will be run only on the `web` instance.

{{< note >}}

> Cron log output is captured in the at `/var/log/cron.log`.  See the [Log page]({{< relref "/development/logs/md" >}}) for more information on logging.

Reviewer: ```suggestion
Cron log output is captured in the at `/var/log/cron.log`.  See the [Log page]({{< relref "/development/logs.md" >}}) for more information on logging.
```

If an application defines both a `web` instance and a `worker` instance, cron tasks will be run only on the `web` instance.

> > Different from the application, the Cron job log is on the `/var/log/cron.log`.

Reviewer 2: ```suggestion
{{< note >}}
Cron logs can be found within the application container at `/var/log/cron.log`.
{{< /note >}}
```

Reviewer 2: Slight re-wording, and uses the note shortcode

> > Different from the application, the Cron job log is on the `/var/log/cron.log`.

Reviewer: ```suggestion
Cron log output is captured in the `/var/log/cron.log` file.  See the [Log page]({{< relref "/development/logs/md" >}}) for more information on logging.
```

Author: Done
